ocs_ci/ocs/hsbench.py

```diff
@@ -42,6 +42,20 @@ def __init__(self):
         self.result = self.hsbench_cr["output_file"]
         self.hsbench_dir = mkdtemp(prefix="hsbench-")
 
+    def _get_bucket_name(self, bucket_num):
+        """
+        Get bucket name from bucket number.
+        Args:
+            bucket_num (int): Number of bucket
+        Returns:
+            bucket_name (str): Name of bucket
```

Comment:
```suggestion
            str: Name of bucket
```
Documenting return type only